src/pages/details/awsBreakdown/awsBreakdown.tsx

```diff
@@ -61,9 +61,9 @@ const mapStateToProps = createMapStateToProps<
       time_scope_value: -1,
       resolution: 'monthly',
       limit: 3,
-      ...(query.filter.account && {['account']: query.filter.account})
+      ...(query && query.filter && query.filter.account && {['account']: query.filter.account})
```

Comment:
nitpick: I believe `query && query.filter_by` will do the same. But if ternary operation is more readable to you then keep it.

Reply:
The problem was that `query` can be undefined -- the code dies here without. First need to check if `query` exists, then check if `query.filter` exists.